Core/clim-basic/text-formatting.lisp

```diff
@@ -128,3 +128,88 @@
                (margin))
            (invoke-with-temporary-page ,stream #',continuation :margins ,margins ,@args))))))
 
+
+;;; Mixin is used to store text-style and ink when filling-output it is invoked.
+
+(defclass %filling-output-graphics-state (gs-ink-mixin gs-text-style-mixin)
+  ()
+  (:default-initargs :ink +foreground-ink+ :text-style *default-text-style*))
+
+(defclass filling-output-mixin (gs-ink-mixin gs-text-style-mixin)
+  ((lbs :accessor line-break-strategy :initarg :line-break-strategy
```

Comment:
When looking at the list of slots in the debugger, seeing a list of names of the form `LBS`, `ALB`, etc, can be confusing. How about longer, more informative names here?

Reply:
slots have documentation, so this should impose a problem. slot-names are purely internal (even more- not used, we access them only via well-defined accessors).

Reply:
I mentioned the debugger (the inspector, actually) which is where I usually see the slot names. If they are descriptive it helps when looking at object instances.

It's not a major issue, but my opinion was merely that since the name isn't actually used anywhere, it doesn't hurt to have them descriptive.

Reply:
imo it is too verbose and will break the indentaiton (which is nicely done with first keywords starting at the same column). I've used short names for aesthetical reasons.